nodes/cma/nodes-cma-autoscaling-custom-rn.adoc

```diff
@@ -30,23 +30,48 @@ The following table defines the Custom Metrics Autoscaler Operator versions for
 |{product-title} version
 |General availability
 
-|2.10.1-267
+|2.11.2
 |4.13
 |General availability
 
-|2.10.1-267
+|2.11.2
 |4.12
 |General availability
 
-|2.10.1-267
+|2.11.2
 |4.11
 |General availability
 
-|2.10.1-267
+|2.11.2
 |4.10
 |General availability
 |===
 
+[id="nodes-pods-autoscaling-custom-rn-2112_{context}"]
+== Custom Metrics Autoscaler Operator 2.11.2-311 release notes
+
+This release of the Custom Metrics Autoscaler Operator 2.11.2-311 provides new features and bug fixes for running the Operator in an {product-title} cluster. The components of the Custom Metrics Autoscaler Operator 2.11.2-311 were released in link:https://access.redhat.com/errata/RHBA-2023:4089[RHBA-2023:4089].
```

Comment:
@mburke5678 not sure if you intended that link to be to the errata or not. When I look on the portal for downloading the Custom Metric Autoscaler, I get nothing, at this link: https://access.redhat.com/downloads/content/704/ver=2/rhel---8/2/x86_64/packages (but the Errata which you link to IS there.) This isn't a correction, I'm just asking you because I expected a download link not the errata link.

Reply:
@GroceryBoyJr No, it is a correction! I had an older errata link there. Thanks for catching this!